package-json-achema.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Is `achema` in the filename meant to be `schema`?

Reply:
Yep, my bad!